packages/store-indexer/bin/postgres-indexer.ts

```diff
@@ -39,7 +39,7 @@ const database = drizzle(postgres(env.DATABASE_URL));
 
 if (await shouldCleanDatabase(database, chainId)) {
   console.log("outdated database detected, clearing data to start fresh");
-  cleanDatabase(database);
+  await cleanDatabase(database);
```

Comment:
oooof, wonder if we should enable a linter for this

feels like we have fewer instances of calling an async function and not awaiting right away and can add a lint ignore for the line we use that approach explicitly

Reply:
noted here: